cmd/bot/run.go

```diff
@@ -76,65 +76,80 @@ func runCommand(ctx *cli.Context) error {
 // and it will finalize the withdrawal when the challenge time window has passed.
 func ProcessBotDelegatedWithdrawals(ctx context.Context, log log.Logger, db *gorm.DB, l1Client *core.ClientExt, l2Client *core.ClientExt, cfg core.Config) {
 	ticker := time.NewTicker(3 * time.Second)
-	processor := core.NewProcessor(log, l1Client, l2Client, cfg)
-	limit := 1000
-
 	for {
 		select {
 		case <-ticker.C:
 		case <-ctx.Done():
 			return
+		default:
+			ProcessUnprovenBotDelegatedWithdrawals(ctx, log, db, l1Client, l2Client, cfg)
+			ProcessUnfinalizedBotDelegatedWithdrawals(ctx, log, db, l1Client, l2Client, cfg)
```

Comment:
should be in the `case <- ticker.C` instead of `default`

Reply:
Thanks for pointing out this bug.
I fix it at [c5fcd17](https://github.com/bnb-chain/opbnb-bridge-bot/pull/7/commits/c5fcd17cf872dbea790ee802283b7e0bee5792b2), please take a look.